pkg/api/v1beta3/conversion.go

```diff
@@ -1858,6 +1859,7 @@ func init() {
 				}
 			}
 			out.RestartPolicy = RestartPolicy(in.RestartPolicy)
+			out.TerminationGracePeriodSeconds = in.TerminationGracePeriodSeconds
```

Comment:
If this field isn't added to v1beta1 and v1beta2, it will break anyone using those versions as the storage version, which is unlikely but possible. Unless there's a compelling reason not to, please just add the field to the other versions.